### PR TITLE
Rest API updates for all services

### PR DESCRIPTION
**Pull request by Garima-msft**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1 [hunk 1]
```diff
@@ -167,20 +167,36 @@ class SVTResourceResolver: AzSKRoot {
...
-            $organizationId = ($responseObj[0].dataProviders."ms.vss-features.my-organizations-data-provider".organizations | Where-Object {$_.name -eq $this.organizationName}).id
-            $inputbody = $null;
-            Remove-Variable inputbody;
+        try { 
+            if (-not [string]::IsNullOrWhiteSpace($env:RefreshToken) -and -not [string]::IsNullOrWhiteSpace($env:ClientSecret))
+            {
+                $apiURL = "https://app.vssps.visualstudio.com/_apis/accounts"
+                $responseObj = "";
+                
+                $responseObj = [WebRequestHelper]::InvokeGetWebRequest($apiURL) ;
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1:176]: do we realy need this api call to get organizationid? we already called org api in SVTResourceResolver and get orgId, cant we use same one?

File: src/AzSK.ADO/Framework/Helpers/ContextHelper.ps1 [hunk 2]
```diff
@@ -210,6 +211,24 @@ class ContextHelper {
         }
         [ContextHelper]::currentContext = $contextObj
     }
+    
+    hidden static [string] GetOAuthUserIdentity($accessToken)
+    {
+        $apiURL = "https://dev.azure.com/safetitestvso/_apis/connectionData"
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Helpers/ContextHelper.ps1:217]: safetitestvso???

File: src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1 (hunk 1 above)
Comment (Garima-msft) [reply src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1:176]: This is the only api call in entire module where we re getting orgid, correct me if i am wrong

Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1:176]: ya  its  same place

File: src/AzSK.ADO/Framework/Helpers/ContextHelper.ps1 (hunk 2 above)
Comment (Garima-msft) [reply src/AzSK.ADO/Framework/Helpers/ContextHelper.ps1]: Fixed

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Release.ps1 [hunk 3]
```diff
@@ -440,117 +479,96 @@ class Release: ADOSVTBase
...
                     $controlResult.AdditionalInfo += "Total number of identities that have access to release pipeline: " + ($accessList | Measure-Object).Count;
                     $controlResult.AdditionalInfo += "Total number of user identities that have access to release pipeline: " + (($accessList | Where-Object {$_.IdentityType -eq 'user'}) | Measure-Object).Count;
                     $controlResult.AdditionalInfo += "Total number of group identities that have access to release pipeline: " + (($accessList | Where-Object {$_.IdentityType -eq 'group'}) | Measure-Object).Count;
+  
                 }
                 else
                 {
-                    $controlResult.AddMessage([VerificationResult]::Passed,"No identities have been explicitly provided with RBAC access to [$($this.ResourceContext.ResourceName)] pipeline other than release pipeline owner and default groups");
+                    $controlResult.AddMessage([VerificationResult]::Passed,"No identities have been explicitly provided with RBAC access to [$($this.ResourceContext.ResourceName)] other than release pipeline owner and default groups");
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Release.ps1:552]: message update

**Comment by Garima-msft** [comment]
Raised separate PR

**Pull request closed without merging**